packages/shared/src/components/comments/MainComment.tsx

```diff
@@ -61,15 +65,21 @@ export default function MainComment({
 
   return (
     <section
-      className="flex flex-col items-stretch rounded-24 border border-theme-divider-tertiary scroll-mt-16"
+      className={classNames(
+        'flex flex-col items-stretch rounded-24 border border-theme-divider-tertiary scroll-mt-16',
+        className,
+      )}
       data-testid="comment"
     >
       {!editProps && (
         <CommentBox
           {...props}
           comment={comment}
           parentId={comment.id}
-          className={{ container: 'border-b' }}
+          className={{
+            container: 'border-b',
+            ...commentBoxClassName,
```

Comment:
This won't do much right?
className here is a object, we read it as `className.container` etc.
Not sure you need it altogether, if so you need to adjust the passing.